ecl/hql/hqlexpr.cpp

```diff
@@ -9873,6 +9873,8 @@ IHqlScope * CHqlVirtualScope::deriveConcreteScope()
             IHqlExpression *cur = symbols.mapToValue(&iter.query());
             OwnedHqlExpr mapped = replacer.transform(cur);
             OwnedHqlExpr rebound = cloneFunction(mapped);
+            if (rebound->isFunction() && rebound->queryChild(1)->numChildren() != 0 && (mapped == rebound), false)
```

Comment:
This change does not look like the others (and the && ... , false is rather odd?

Reply:
no, it wasn't meant to be included.  It was an internal consistency check part of HPCC-23091.